oauth

```diff
@@ -0,0 +1 @@
+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
```

Comment:
Nice, and thank you github for doing the immediate revoke -- even faster than I could goto https://github.com/settings/tokens and click the "Delete" key.